nestoria-project/src/app/modules/property-search-page/property-search.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="nav">
+  <div>
+    <h2>Property Cross</h2>
+  </div>
+  <div>
+    <app-button [id]="'button-favor'" label="Faves" click="navigateToFavor()"></app-button>
+  </div>
+</div>
+
+<div>
+  Use the form below to search for houses to buy. You can search by place-name, postcode, or click 'My location', to
+  search in your current location!
+</div>
+<br>
+<div class="search-block">
+  <div class="search-item"><input type="text" class="search-input" placeholder="Search..."></div>
+  <div class="buttons">
+    <div class="button-wrapper">
+      <app-button [id]="'button-search'" label="Go" onclick="doSearch()"></app-button>
+    </div>
+    <div class="button-wrapper">
+      <app-button [id]="'button-location'" label="My location" onclick="defineLocation()"></app-button>
```

Comment:
I see thing each better to move in config file like - app.constants and create const property with this values - 'button-search', 'button-location' and use them